lib/funky/html/parser.rb

```diff
@@ -29,6 +30,7 @@ def extract_views_from(html)
       def extract_likes_from(html)
         html.match(/"likecount":(\d+),"likecountreduced"/)
         html.match(/likecount:(\d+),likecountreduced/) if $1.nil?
+        html.match(%r[{localized_name:"Like",reaction_type:"LIKE",id:"1635855486666999",key:1},reaction_count:(\d+)}])
```

Comment:
This seems super brittle. Is matching on a digit in the `id` too flexible? 🤔

Reply:
I tried multiple videos and it worked for them. I'll fix it if there's another problem.

```
Funky::Video.find('313477029297950')
Funky::Video.find('486529068458871')
Funky::Video.find('1118059165038879')
Funky::Video.find('217791842435207')
```